### PR TITLE
Fix date formatting in the app

### LINKED ISSUE #120: Fix date formatting in the app
### TODO
This task has two steps:
- [x] Fix date formatting
- [x] If the month in the end date is the same as the current month, display "current" string as the end date 

### TIPS
Currently, the dates have no formatting, they are displayed in a very inconvenient format. 
Screenshot 2020-11-02 at 21 45 42
Use already installed `dayjs` library to format the dates like on the mockups
Screenshot 2020-11-02 at 21 58 55

### PR DESCRIPTION
**Pull request by mecies**
## Description

Added date formatting and displaying "current" if the date is the current month
Also refactored dates in the blog page and portfolio page

Closes #120 

## Preview

Following dates:
`2019-11-02T20:06:48.583Z - 2020-11-01T20:06:48.592Z`
`2020-02-02T20:09:54.359Z - 2020-09-01T19:09:54.371Z`
`2017-11-02T20:11:46.025Z - 2018-11-02T20:12:59.850Z`

Are displayed as such:

Screenshot 2020-11-07 at 18 51 15

### CODE REVIEW

File: src/utils/date.ts [hunk 1]
```diff
@@ -0,0 +1,24 @@
+import dayjs from 'dayjs';
+
+export const dateFormatMonth = 'MMMM YYYY';
```
Comment (KonradSzwarc) [inline src/utils/date.ts]: You're not using this constant anywhere, so maybe there is no need for exporting it?

File: src/utils/date.ts [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+  const currentMonth = dayjs().format(dateFormatMonth);
+
+  return dayjs(date).format(dateFormatMonth) === currentMonth;
+};
+
+export const formatDate = (date: string, format: DateFormat) => {
+  return format === 'day' ? dayjs(date).format(dateFormatDay) : dayjs(date).format(dateFormatMonth);
+};
+
+export const displayCurrentIfCurrentDate = (date: string, format: DateFormat) => {
```
Comment (KonradSzwarc) [inline src/utils/date.ts]: Maybe it would be cleaner if you put the logic of this function into `formatDate`?

**Review by KonradSzwarc — CHANGES_REQUESTED** [review]

File: src/utils/date.ts (hunk 1 above)
Comment (mecies) [reply src/utils/date.ts]: Good catch 🦅 👁️

File: src/utils/date.ts (hunk 2 above)
Comment (mecies) [reply src/utils/date.ts]: I gave it a shot 😄

**Review by KonradSzwarc — APPROVED** [review]

**Review by KlaraGajaszek — APPROVED** [review]

**Pull request merged**